### ISSUE TITLE
IllegalArgumentException (JMEter-CCFinder)

### ISSUE THREAD

**Issue by dmazinanian** [tsantalis/JDeodorant#30]
Clone # 565 (Group 4854, Pair 1-2): org.apache.jmeter.protocol.jdbc.config.DataSourceElementBeanInfo#DataSourceElementBeanInfo (row 1580) and org.apache.jmeter.protocol.jdbc.sampler.JDBCSamplerBeanInfo#JDBCSamplerBeanInfo (row 1581)

```
java.lang.IllegalArgumentException: Invalid identifier : >class<
    at org.eclipse.jdt.core.dom.SimpleName.setIdentifier(SimpleName.java:195)
    at org.eclipse.jdt.core.dom.AST.newSimpleName(AST.java:2202)
    at gr.uom.java.jdeodorant.refactoring.manipulators.ExtractCloneRefactoring.extractClone(ExtractCloneRefactoring.java:721)
    at gr.uom.java.jdeodorant.refactoring.manipulators.ExtractCloneRefactoring.apply(ExtractCloneRefactoring.java:331)
    at gr.uom.java.jdeodorant.refactoring.manipulators.ExtractCloneRefactoring.checkFinalConditions(ExtractCloneRefactoring.java:4833)
    at ca.concordia.jdeodorant.eclipse.commandline.Application.testRefactoring(Application.java:667)
    at ca.concordia.jdeodorant.eclipse.commandline.Application.start(Application.java:184)
    at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:673)
    at org.eclipse.equinox.launcher.Main.basicRun(Main.java:610)
    at org.eclipse.equinox.launcher.Main.run(Main.java:1519)
    at org.eclipse.equinox.launcher.Main.main(Main.java:1492)
```

**Comment by tsantalis** [comment]
Fixed in commit [3dfa1f4bf465aa4d81a0bc9d7de7b5b5f296bb41](https://github.com/tsantalis/JDeodorant/commit/3dfa1f4bf465aa4d81a0bc9d7de7b5b5f296bb41)

**Comment by dmazinanian** [comment]
I guess this is happening again in JRuby-CCFinder:

Clone # 916 (Group 6009, Pair 1-2): org.jruby.parser.LocalStaticScope#assign (row 2281) and org.jruby.parser.BlockStaticScope#assign (row 2282)

**Comment by tsantalis** [comment]
Fixed in commit [16a063333511c18a76ef038ba0f3c6064e77b5c7](https://github.com/tsantalis/JDeodorant/commit/16a063333511c18a76ef038ba0f3c6064e77b5c7)
